packages/Icon/src/Icon.spec.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import Icon from '.';
+
+afterAll(cleanup);
+
+// Test here
```

Comment:
Let's make one placeholder test that just asserts something minimal. Not sure what tbh....if you can't think of one either then I guess we can skip :\

Reply:
I will add tests as a part of this PR – this was just a placeholder for the moment, and wasn't intending for this to go into master as it is.